dapps/profile.vue/src/i18n/de.json

```
@@ -189,6 +189,17 @@
       "registration": "Registrierungsdaten"
     },
     "not-permitted-view": "Sie haben keine Rechte, diese Daten für die ausgewählte Identität anzusehen.",
+    "security-info": {
+      "encryption-key": {
+        "exported": "Encryption Key exportiert und in die Zwischenablage gelegt",
+        "title": "Encryption Key exportieren"
+      },
+      "private-key": {
+        "exported": "Private Key exportiert und in die Zwischenablage gelegt",
+        "title": "Private Key exportieren"
+      },
```

I think the user doesn't care about what happens in the background, so I would change the wording a bit:


```suggestion
      "encryption-key": {
        "exported": "Encryption-Key in die Zwischenablage kopiert",
        "title": "Encryption Key kopieren"
      },
      "private-key": {
        "exported": "Private Key in die Zwischenablage kopiert",
        "title": "Private Key kopieren"
      },
```